scene/gui/scroll_bar.cpp

```diff
@@ -665,6 +672,7 @@ ScrollBar::ScrollBar(Orientation p_orientation) {
 	highlight = HIGHLIGHT_NONE;
 	custom_step = -1;
 	drag_node = NULL;
+	drag_node_enabled = true;
```

Comment:
This can be initialized directly on declaration in the header (same for other members, feel free to port them).

Reply:
Moved all initializers to respective header file.